iosApp/iosAppUITests/bottomNavigationUITests.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  iosAppUITests.swift
+//  iosAppUITests
+//
+//  Created by Jose Mateo on 1/23/22.
+//  Copyright © 2022 orgName. All rights reserved.
+//
+
+import XCTest
+import iosApp
+
+class bottomNavigationUITests: XCTestCase {
```

Comment:
Swift naming convention for classes is Capitalized. (Same for file name)

```suggestion
class BottomNavigationUITests: XCTestCase {
```

Reply:
Agree I'll update these. It was weird that Xcode autogenerated the test  files with a lower case letter before I renamed it